### ISSUE TITLE
physdim context is missing

### ISSUE THREAD

**Issue by VladimirAlexiev** [IIIF/iiif.io#625]
http://iiif.io/api/annex/service/physdim/1/context.json is missing. This makes example in http://iiif.io/api/image/2.0/#image-information invalid JSONLD RIOT
```
>riot --validate image-iiif.jsonld
19:18:48 ERROR riot :: invalid remote context: http://iiif.io/api/annex/service/physdim/1/context.json
```
Would be nice if you can add this file. Thanks!

**Comment by VladimirAlexiev** [comment]
I'm guessing this context should be
```json
{
  "@context": {
    "iiif" :   "http://iiif.io/api/image/2#",
    "xsd" :    "http://www.w3.org/2001/XMLSchema#",

    "physicalScale" : {"@id" : "iiif:physicalScale", "@type": "xsd:float" },
    "physicalUnits" : {"@id" : "iiif:physicalUnits", "@type": "xsd:string" }
  }
}
```
At least RIOT is now quiet.

**Comment by azaroth42** [comment]
Typo for .../annex/services/physdim/... The context file is in the right place, just the link is wrong in /annex/services/index.html :blush: 

Fix in branch: http://friday_issues.iiif.io/api/annex/services/

**Comment by zimeon** [comment]
It seems that there is still a problem even when the link is fixed. <http://iiif.io/api/annex/services/physdim/1/context.json> has:

```json
{
	"@context": {
    	"iiif" :   "http://iiif.io/api/presentation/{{ site.presentation_api.latest.major }}#",
    	"xsd" :    "http://www.w3.org/2001/XMLSchema#",
		"physicalScale": {"@id": "iiif:physicalScale", "@type": "xsd:float"},
		"physicalUnits": {"@id": "iiif:physicalUnits", "@type": "xsd:string"}
	}
}
```

which includes unexpanded Jekyll. And perhaps I need an explanation of then intention for somehow "auto-versioning" the context anyway. Seems to me like we probably want to have a specific physdim context for a specific prezi api major version?

**Comment by jpstroop** [comment]
Jekyll isn't expanding the variable because the YAML header is missing. It can be completely empty, but it has to be there, i.e., file content:

```
---
---
{
	"@context": {
    	"iiif" :   "http://iiif.io/api/presentation/{{ site.presentation_api.latest.major }}#",
    	"xsd" :    "http://www.w3.org/2001/XMLSchema#",
		"physicalScale": {"@id": "iiif:physicalScale", "@type": "xsd:float"},
		"physicalUnits": {"@id": "iiif:physicalUnits", "@type": "xsd:string"}
	}
}
```

will yield ` "iiif": "http://iiif.io/api/presentation/2#"`

I'd put in a PR, but I don't know where we are with having mergegate solved, and don't want to add further confusion.

**Comment by azaroth42** [comment]
So I put this in and it didn't work. It still stays unexpanded :(

I'll hard code the 2, which would address Simeon's issue of not dynamically changing the physdim context.

**Comment by zimeon** [comment]
<http://friday_issues.iiif.io/api/annex/services/#physical-dimensions> :+1:

**Comment by jpstroop** [comment]
Looks good, but the variable expansion should absolutely work:

https://github.com/IIIF/iiif.io/blob/master/source/api/image/2/level2.json#L1-L5

makes:

http://iiif.io/api/image/2/level2.json

**Comment by azaroth42** [comment]
Yeah, PEBKAC for it not working.  That said, I think I prefer static as per
head of the branch.

R

On Mon, Dec 7, 2015 at 4:01 PM, Jon Stroop <notifications@github.com> wrote:

> Looks good, but the variable expansion should absolutely work:
>
>
> https://github.com/IIIF/iiif.io/blob/master/source/api/image/2/level2.json#L1-L5
>
> makes:
>
> http://iiif.io/api/image/2/level2.json
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IIIF/iiif.io/issues/625#issuecomment-162549154>.
>



-- 
Rob Sanderson
Information Standards Advocate
Digital Library Systems and Services
Stanford, CA 94305

**Comment by jpstroop** [comment]
Are we going to change them all then? Sort of makes sense, I guess, since they should reflect the name of the directory, which shouldn't change.

**Comment by zimeon** [comment]
I also think the number should be static

**Comment by jpstroop** [comment]
Yes, in fact it would be bad if they did change.

**Comment by jpstroop** [comment]
Static context URIs noted in #633 and solved by #635
